docs/signals/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Snowplow Signals"
+sidebar_position: 8
+description: "An overview of Signals concepts."
+sidebar_label: "Signals"
+---
+
+Snowplow Signals is a personalization engine built on Snowplow’s behavioral data pipeline. The Profile API, hosted in your BDP cloud allows you to create, manage and access user attributes by using the Signals SDKs.
+
+![](./images/signals.png)
```

Comment:
move this diagram into the new Signals components subheading below, replace it with the fancy official one that matches the docs homepage architecture diagram